generator/uscore/templates/sequence.rb.erb

```diff
@@ -9,14 +9,18 @@ module Inferno
 
       test_id_prefix '<%=test_id_prefix%>'
 
-      requires :token<%=", :patient_id" unless delayed_sequence%>
+      requires :token<%=", :patient_ids" unless delayed_sequence%>
       conformance_supports :<%=resource%><%="
       delayed_sequence" if delayed_sequence%>
 <%=search_validator%>
       details %(
         The #{title} Sequence tests `#{title.gsub(/\s+/, '')}` resources associated with the provided patient.
       )
 
+      def patient_ids
```

Comment:
Should this go in SequenceBase? I know putting more stuff in SequnceBase is unfortunate, but I think it makes sense since this method seems necessary to support the new field in TestingInstance.

Reply:
I don't think so -- to me it is a matter of scope. Should all tests everywhere need access to this method?  I don't think so.  Yeah it's unfortunate that all testing instances will have this field, but hopefully we can fix that at some point.